l3_dump.py

```diff
@@ -31,11 +31,14 @@
 
 with open(sys.argv[1], 'rb') as file:
     data = file.read(32)
-    idx, fibase, _, _ = struct.unpack('<QQQQ', data)
+    idx, loc, fibase, _, _ = struct.unpack('<iiQQQ', data)
```

Comment:
Unpack the 32-byte struct as {uint32, uint32, uint64, uint64, uint64}, which will now unpack the newly added `loc` ID value also.